<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="goods" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>321</width>
      <height>581</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="widget11" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
     <widget class="QLabel" name="image11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/11.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget12" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLabel" name="image12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/12.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget13" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
     <widget class="QLabel" name="image13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/13.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget21" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>21</string>
      </property>
     </widget>
     <widget class="QLabel" name="image21">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/21.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget22" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item22">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>22</string>
      </property>
     </widget>
     <widget class="QLabel" name="image22">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/22.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget23" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>23</string>
      </property>
     </widget>
     <widget class="QLabel" name="image23">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/23.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget33" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item33">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>33</string>
      </property>
     </widget>
     <widget class="QLabel" name="image33">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/33.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget31" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item31">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>31</string>
      </property>
     </widget>
     <widget class="QLabel" name="image31">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/31.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget32" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item32">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>32</string>
      </property>
     </widget>
     <widget class="QLabel" name="image32">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/32.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget41" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item41">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>41</string>
      </property>
     </widget>
     <widget class="QLabel" name="image41">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/41.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget42" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>440</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item42">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>42</string>
      </property>
     </widget>
     <widget class="QLabel" name="image42">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/42.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget43" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>440</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <widget class="QLabel" name="item43">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>43</string>
      </property>
     </widget>
     <widget class="QLabel" name="image43">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/43.png</pixmap>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="payment" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>241</width>
      <height>721</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="toBePaid">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>157</red>
           <green>240</green>
           <blue>88</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>142</red>
           <green>203</green>
           <blue>142</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>157</red>
           <green>240</green>
           <blue>88</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>142</red>
           <green>203</green>
           <blue>142</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>157</red>
           <green>240</green>
           <blue>88</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="yourMoney">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>217</red>
           <green>217</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QPushButton" name="add1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </widget>
    <zorder>yourMoney</zorder>
    <zorder>toBePaid</zorder>
    <zorder>widget</zorder>
   </widget>
   <widget class="QWidget" name="purchase" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>610</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>171</red>
          <green>171</green>
          <blue>171</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
